src/Tizen.NUI/src/public/Accessibility/Accessibility.cs

```diff
@@ -32,7 +32,7 @@ namespace Tizen.NUI.Accessibility
     // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
     [SuppressMessage("Microsoft.Design", "CA1724: Type names should not match namespaces")]
     [EditorBrowsable(EditorBrowsableState.Never)]
-    public class Accessibility
+    public class Accessibility : IDisposable
```

Comment:
Accessibility is also a singleton class like DisposableQueue.

Reply:
Done